### DISCUSSION TITLE
How to combine multiple folder fasta output into one folder

### DISCUSSION THREAD

**Discussion by mirzaabidpp** [kblin/ncbi-genome-download#185] [Q&A]
Hi, 
Thank you very much for this amazing tool. I was struggling to download my desired genomes from NCBI then I came across this tool which helped me a lot and made my day. As this tool downloads genome files in separate folders but I want to combine all separate output in one folder for merging them according to my need. Please help me to solve this problem as the list of downloaded genomes is more than 1000 and is difficult to combine them manually.

Regards

**Comment by kblin** [comment]
Hi, thanks for your patience. It seems like something ate my github notification on this one.
I'd go with a small shell script for this task. Assuming you're interested in all files ending in `.gbff.gz` in the `refseq` folder and want to put them into a folder called `everything`, you'd do something like this:

```bash
for filename in $(find refseq -iname "*.gbff.gz"); do
    cp $filename everything/
done;
```
Hope this helps,
Kai

**Comment by kblin** [comment]
I've just added a `--flat-output` parameter to `ncbi-genome-download` that allows you to keep all downloaded files in one folder.
